CMakeLists.txt

```diff
@@ -503,8 +512,12 @@ if(HARDENING)
     try_append_linker_flag("/HIGHENTROPYVA" TARGET hardening_interface)
     try_append_linker_flag("/NXCOMPAT" TARGET hardening_interface)
   else()
-    target_compile_options(hardening_interface INTERFACE
-      $<$<NOT:$<CONFIG:Debug>>:-U_FORTIFY_SOURCE -D_FORTIFY_SOURCE=3>
+    add_library(hardening_interface_nondebug INTERFACE)
+    target_link_libraries(hardening_interface INTERFACE
+      $<$<NOT:$<CONFIG:Debug>>:hardening_interface_nondebug>
+    )
+    target_compile_options(hardening_interface_nondebug INTERFACE
```

Comment:
Why has this moved from `hardening_interface` to `hardening_interface_nondebug` ? Not actually sure why we'd have different hardening options depending on wether you're debugging?

Reply:
This also seems to add even more complexity to printing the summary, which is basically already impossible to follow.